test/e2e/service_options_test.go

```diff
@@ -72,6 +72,11 @@ func TestServiceOptions(t *testing.T) {
 	validateServiceMinScale(r, "svc2", "1")
 	validateServiceMaxScale(r, "svc2", "3")
 
+	t.Log("create and validate service with scale options ")
+	serviceCreateWithOptions(r, "svc2a", "--scale", "5")
```

Comment:
I wonder if you want to try to change scale value for both direction up and down... Thanks.